src/scripts/macros/hotbar.ts

```diff
@@ -7,6 +7,7 @@ import { ChatMessagePF2e } from "@module/chat-message/document.ts";
 import { createUseActionMessage } from "@module/chat-message/helpers.ts";
 import { MacroPF2e } from "@module/macro.ts";
 import { objectHasKey } from "@util";
+import { eventToRollMode } from "@util/sheet.ts";
```

Comment:
Would these be importable from the above?

Reply:
Interestingly enough, currently no. That said I think the plan is to move everything in @util/sheet.ts to @module/sheet/helpers.ts. Our sheet helpers are spread out and you preferred the second one.